services/frontend-service/src/ui/utils/store.test.tsx

```diff
@@ -452,3 +462,61 @@ describe('Rollout Status', () => {
         });
     });
 });
+
+describe('Test maxActions', () => {
+    type TestDataStore = {
+        name: string;
+        inputActionsLen: number;
+        expectedLen: number;
+        expectedShowError: boolean;
+    };
+
+    const testdata: TestDataStore[] = [
+        {
+            name: 'below limit',
+            inputActionsLen: 99,
+            expectedLen: 99,
+            expectedShowError: false,
+        },
+        {
+            name: 'at limit',
+            inputActionsLen: 100,
+            expectedLen: 100,
+            expectedShowError: false,
+        },
+        {
+            name: 'over limit',
+            inputActionsLen: 101,
+            expectedLen: 100,
```

Comment:
I am a bit hesitant here. Because if the snackbar errors, I am actually expecting only a error and not an action list of size, for example, 100. Or am I missing something?

Reply:
the test puts in 101 actions one by one. So at the end there will be only 100 items in it, because the last one is not allowed to be added.